### PR TITLE
Access all of the Group IDs associated with a Project or Subproject

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/ProjectRestController.java [hunk 1]
```diff
@@ -161,5 +171,16 @@ public void setRequestManager(RequestManager requestManager) {
     mapper.getSerializationConfig().addMixInAnnotations(User.class, UserInfoMixin.class);
     return mapper.writeValueAsString(lp);
   }
-  
+
+  @RequestMapping(value = "{id}/groups", method = RequestMethod.GET, produces = { "application/json" })
+  @ResponseBody
+  public Collection<Integer> getSubprojectGroups(@PathVariable("id") Long id, UriComponentsBuilder uriBuilder, HttpServletResponse response)
+      throws IOException {
```
Comment (callunity) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/ProjectRestController.java]: This be 'getProjectSampleGroups' or something similarly Project-related

**Comment by callunity** [comment]
:+1: after change

**Comment by apmasell** [comment]
Fixed.

**Comment by djcooke** [comment]
:+1:

**Pull request merged**